editor/editor_toaster.cpp

```diff
@@ -441,14 +449,31 @@ void EditorToaster::_popup_str(String p_message, Severity p_severity, String p_t
 		main_button->queue_redraw();
 	}
 
-	// Retrieve the label back then update the text.
-	Label *label = Object::cast_to<Label>(control->get_child(0)->get_child(0));
+	// Retrieve the label back, then update the text.
+	Label *label = Object::cast_to<Label>(control->get_child(0)->get_child(0)->get_child(0));
```

Comment:
This is really hacky. Would be better to keep a pointer as member variable, like we do everywhere else.

Reply:
That would mean I need to add both to the Toast struct.
I cannot simply keep the reference to the `label` / `count_label` variable before, as I need to find it again in the toast list when the goal is only to increase the count.

But yeah, it's not a big deal, I can implement it that way too.

Reply:
Could it use a node unique name at least? This kind of code is asking for breakage and not a very good example of best UI practices :)